eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -12,76 +9,70 @@ parameters:
-                  inputs:
-                    filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
-                    arguments: >
-                      -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
-                      -workingDirectory $(System.DefaultWorkingDirectory)
-                      -packageRepository 'Nuget'
-                      -releaseSha '$(Build.SourceVersion)'
-                      -repoOwner 'Azure'
-                      -repoName 'azure-sdk-for-net'
-                    pwsh: true
-                  env:
-                    GH_TOKEN: $(azuresdk-github-pat)
-                  condition: and(succeeded(), ne(variables['SkipCreateTagAndGitRelease'], 'true'))
+      - task: Powershell@2
+        displayName: 'Verify Package Tags and Create Git Releases'
+        inputs:
+          filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
+          arguments: >
+            -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
+            -workingDirectory $(System.DefaultWorkingDirectory)
+            -packageRepository 'Nuget'
+            -releaseSha '$(Build.SourceVersion)'
+            -repoOwner 'Azure'
+            -repoName 'azure-sdk-for-net'
+          pwsh: true
+        env:
+          GH_TOKEN: $(azuresdk-github-pat)
+        condition: and(succeeded(), ne(variables['SkipCreateTagAndGitRelease'], 'true'))
 
-                - ${{ if eq(parameters.ShouldPublish, true) }}:
-                  - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
-                    parameters:
-                      PackagesPath: '$(PIPELINE.WORKSPACE)/packages'
-                      BuildToolsPath: '${{ parameters.BuildToolsPath }}'
+      - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
+        parameters:
+          PackagesPath: '$(PIPELINE.WORKSPACE)/packages'
+          BuildToolsPath: '${{ parameters.BuildToolsPath }}'
 
-                  - task: MSBuild@1
-                    displayName: 'Upload Symbols'
-                    inputs:
-                      solution: '${{ parameters.BuildToolsPath }}/tools/symboltool/SymbolUploader.proj'
-                      msbuildArguments: >
-                        /p:PackagesPath=$(PIPELINE.WORKSPACE)/packages
-                        /p:MSPublicSymbolsPAT=$(azuresdk-microsoftpublicsymbols-devops-pat)
-                        /p:MSSymbolsPAT=$(azuresdk-microsoft-devops-pat)
-                        /p:AzureSDKSymbolsPAT=$(azuresdk-azure-sdk-devops-pat)
-                    condition: and(succeeded(), eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net'))
+      - task: MSBuild@1
+        displayName: 'Upload Symbols'
+        inputs:
+          solution: '${{ parameters.BuildToolsPath }}/tools/symboltool/SymbolUploader.proj'
+          msbuildArguments: >
+            /p:PackagesPath=$(PIPELINE.WORKSPACE)/packages/Azure.Template
```

Comment:
Why did you add "Azure.Template" here? Was that leftover from testing?

Reply:
Yes, I used the data plane template package to test it.